baseapp/genesis.go

```diff
@@ -0,0 +1,71 @@
+package baseapp
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"time"
+
+	crypto "github.com/tendermint/go-crypto"
+	cmn "github.com/tendermint/tmlibs/common"
+)
+
+// TODO this is dup code from tendermint-core to avoid dep issues
+// should probably remove from both SDK / Tendermint and move to tmlibs
+//  ^^^^^^^^^^^^^ This is my preference to avoid DEP hell
+// or sync up versioning and just reference tendermint
```

Comment:
lol our deps are in a brutal state :( 

we need to make sure it never happens again. obscene.